app/services/checksum_validator.rb

```diff
@@ -47,7 +50,10 @@ def validate_signature_catalog
     checksum_results.report_results
   end
 
-  private
+  def flag_unexpected_data_content_files
```

Comment:
actually, this needs to check in both "data/content" and "/data/metadata".   manifestsInventory is for top level "manifests" directory, and signatureCatalog is for everything under the "data" directory, no?

(it's confusing because the word "manifest" is similar to the word "metadata" and I get them confused all the time)

I would rename to `flag_unexpected_data_files`